app/build.gradle

```diff
@@ -35,11 +35,12 @@ dependencies {
     implementation "androidx.core:core-ktx:1.7.0"
     implementation "androidx.appcompat:appcompat:1.4.2"
     implementation "com.google.android.material:material:1.6.1"
+    implementation "com.google.android.gms:play-services-tasks:18.0.2"
```

Comment:
Any reason for adding the play services as a dependency?

Reply:
@hovig-theo I'm not sure, that doesn't seem necessary indeed. I'll remove it.